lib/oli_web/templates/pow_invitation/invitation/edit.html.eex

```diff
@@ -0,0 +1,51 @@
+<%= render OliWeb.SharedView, "_form_container.html", Map.put(assigns, :title, value_or(assigns[:title], "Register")) do %>
+<%= form_for @changeset, @action, [as: :user], fn f -> %>
+<%= if @changeset.action do %>
+<div class="alert alert-danger">
+    <p>Oops, something went wrong! Please check the errors below.</p>
+</div>
+<% end %>
+
+<div class="form-label-group">
+    <%= email_input f, Pow.Ecto.Schema.user_id_field(@changeset), class: "form-control " <> error_class(f, Pow.Ecto.Schema.user_id_field(@changeset), "is-invalid"),
+        placeholder: "Email", required: true, autofocus: focusHelper(f, Pow.Ecto.Schema.user_id_field(@changeset)) %>
+    <%= label f, Pow.Ecto.Schema.user_id_field(@changeset), class: "control-label" %>
+    <%= error_tag f, Pow.Ecto.Schema.user_id_field(@changeset) %>
+</div>
+
+<div class="form-label-group">
+    <%= text_input f, :given_name, class: "form-control " <> error_class(f, :given_name, "is-invalid"),
+        placeholder: "First Name", required: true, autofocus: focusHelper(f, :given_name) %>
+    <%= label f, :given_name, class: "control-label" %>
+    <%= error_tag f, :given_name %>
+</div>
+
+<div class="form-label-group">
+    <%= text_input f, :family_name, class: "form-control " <> error_class(f, :family_name, "is-invalid"),
+        placeholder: "Last Name", required: true, autofocus: focusHelper(f, :family_name) %>
+    <%= label f, :family_name, class: "control-label" %>
+    <%= error_tag f, :family_name %>
+</div>
+
+<div class="form-label-group">
+    <%= password_input f, :password, class: "form-control " <> error_class(f, [:password, :password_confirmation], "is-invalid"),
+        placeholder: "New Password", required: true, autofocus: focusHelper(f, [:password, :password_confirmation]) %>
+    <%= label f, :password, class: "control-label" %>
+    <%= error_tag f, :password %>
+</div>
+
+<div class="form-label-group">
+    <%= password_input f, :password_confirmation, class: "form-control " <> error_class(f, :password_confirmation, "is-invalid"),
+        placeholder: "Confirm New Password", required: true, autofocus: focusHelper(f, :password_confirmation) %>
+    <%= label f, :password_confirmation, "Confirm Password", class: "control-label" %>
+    <%= error_tag f, :password_confirmation %>
+</div>
+
+<%= if assigns[:link_account] do %>
+<%= hidden_input f, :link_account, value: @link_account %>
+    <% end %>
+
+<%= submit "Submit", class: "btn btn-md btn-primary btn-block" %>
```

Comment:
I think it would make it clearer about the difference between Submit and Sign in if we rename Submit to something like Create Account

Reply:
Button text switched to "Create Account"